ste/mgr-JobPartTransferMgr.go

```diff
@@ -881,3 +882,10 @@ func (jptm *jobPartTransferMgr) GetDestinationRoot() string {
 	p := jptm.jobPartMgr.Plan()
 	return string(p.DestinationRoot[:p.DestinationRootLength])
 }
+
+func (jptm *jobPartTransferMgr) CanInferContentType() bool {
```

Comment:
Minor: it's more like ShouldInferContentType

Reply:
Yup you're right. Will do the changes